### ISSUE TITLE
AR model reflects environment-image

### ISSUE THREAD

**Issue by paperli** [google/model-viewer#1445]
### Description

I set both the environment-image and skybox-image in the model-viewer, and view the model in AR with Pixel 3. The model reflects the environment-image in WebXR’s AR viewer. It makes no sense because the AR model shall reflect real world environment instead of pre-set env map. Is there an option to prevent environment-image reflection in AR. 
There is no problem either in iOS’s quick look or when set only skybox-image. 

### Version

- model-viewer: v1.0.0

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Correct, and it does this because the WebXR API does not yet expose estimated environment lighting, though it's coming soon. If you select `ar-modes="scene-viewer quick-look"`, those both use estimated lighting, though we've actually gotten even more complaints about that. It's all ML, and sadly nothing is perfect.

**Comment by paperli** [comment]
Thanks for the reply. We did change the at-modes settings to work around the problem. I am still curious about why `skybox-image` doesn’t affect the reflection but `environment-image` dose.

**Comment by elalish** [comment]
`environment-image` is what affects lighting and therefore reflections. It should default to `skybox-image` if only that is specified. Are you seeing different behavior than that?

**Comment by paperli** [comment]
Yes, if I assign only the `skybox-image` property without `environment-image`, the model doesn’t reflect the `skybox-image`. When you compare below images, you can see the env-map is reflected in the first model, which make it darker. But the second one is much brighter. The only difference is with or without `environment-image`.

- Set both `environment-image` and `skybox-image`:
![0290ADF7-92F6-4BD0-8A1B-E5764CC3E2EB](https://user-images.githubusercontent.com/1106951/89587100-bf05a900-d7f5-11ea-8207-c47c2e16d39c.jpeg)


- Set only `skybox-image`:
![3F83D6B1-F115-48FC-BBB4-D75D00C1B15C](https://user-images.githubusercontent.com/1106951/89586978-82d24880-d7f5-11ea-93d0-496592553f15.jpeg)

**Comment by elalish** [comment]
Can you remix the Glitch from our issue template to demonstrate? Also, if you can use colored or sunrise environments just to make them more obvious, that would be helpful. Many other things can contribute to overall brightness.

**Comment by paperli** [comment]
Sorry, I tried again and found that I fired a false alarm. In both with/without environment-image situation, the WebXR's AR mode shows similar behavior. The model reflects either environment-image or skybox-image. I don't exactly know why I saw the different results in the previous test.

However, I share the two set-ups in Glitch:
- With skybox-image and environment-image: https://glitch.com/~chipped-daffy-anaconda
- with skybox-image only: https://glitch.com/~exultant-reinvented-meeting
